packages/round-manager/src/features/api/utils.ts

```diff
@@ -364,7 +365,7 @@ export const pinToIPFS = (obj: IPFSObject): Promise<{ IpfsHash: string }> => {
         ...params.headers,
         "Content-Type": "application/json",
       },
-      body: JSON.stringify({ ...params.body, pinataContent: obj.content }),
+      body: serialize({ ...params.body, pinataContent: obj.content }),
```

Comment:
More context: https://wagmi.sh/core/faq#bigint-serialization

I don't think using the `serialize` / `deserialize` functions from Wagmi is a good idea. I understand why they implemented it but it doesn't apply for our case.

- We sometimes might upload user content, and since with this a string is not really a string, what happens when I answer `#bigint.000` or something to one of the round application questions?
- This was probably designed to encode and decode arbitrary data, in our case we know exactly the shape of the data, we can decode bigints manually, so it's never "lossy" like they mention in the FAQ
- This metadata is part of the protocol, if we agree that we want to represent bigint as `#bigint.xxx` then let's do it, but we're inventing a new convention that depends on the Wagmi library
- We're changing expectations of how JSON encoding / decoding is behaving, we're not expecting a normal JSON anymore, this will break either compatibility or in the future if we move from this


```typescript
type MyObject = {value: bigint};

const objectA: MyObject = { value: 69420n };

const serialized = JSON.stringify({ value: objectA.value.toString() });
const deserialized = JSON.parse(serialized) as { value: string };

const objectB = { value: BigInt(deserialized.value) };
```